### DISCUSSION TITLE
[UserStoriesToRefine] #6 As <user> I want a website that will allow us to better understand our users. I want to better understand what languages are most needed, which topics are the most searched for, what videos are the most useful to out users, and which ones are the most shared with others

### DISCUSSION THREAD

**Discussion by emecas** [emecas/civic-tech-website-consultancy-pilot-translation#28] [General]
### As <user> I want a website that will allow us to better understand our users. I want to better understand what languages are most needed, which topics are the most searched for, what videos are the most useful to out users, and which ones are the most shared with others

**Comment by UnitedByChocolate** [comment]
Level 2 - As a producer I want to be able to share my interests and preferences, so that I can influence future presentations to share information most useful to me.

Level 2 - As a producer I want my preferences tracked in a way that eliminates my own biases

Level 3 - Acceptance Criteria:
1. Allow for random presentation of videos of various topics to be presented each time a new user visits the website
2. Allow producers to suggest follow-on topics
3. Allow the website to be able to detect where producers are located
4. Present the website in the native language of the user based on their "cookie" location
5. Allow for voting (similar to Reddit) that allows to vote up future topics
6. Allow for voting (similar to Reddit) that allows to vote up future presenters and organization representatives

**Comment by UnitedByChocolate** [comment]
**Level 2** - As a producer I want to be able to share my interests and preferences, so that I can influence future presentations to share information most useful to me.
Level 3 - Acceptance Criteria (The Given/When/Then format )
1. Given that I have watched a recorded video, when I am done watching it or am ready to move on, then  I will give suggestions for future topics by voting on various topic themes
2. Given that producers may access the site from all over the world, when they arrive at the site their location will be identified via their unique "cookie", then the website will appear in the language best associated with their location
3. Given that it is important to know what are the most pressing issues for cacao producers, when they access videos, then they will have access to vote (similar to Reddit) topics up and down
4. Given that the most engaging presenters also help cacao producers to engage and learn more, when they access videos, then they will be able to (similar to Reddit) vote up and down the presenters/ organizations that presented in that particular video 

**Level 2** - As a producer I want my preferences tracked in a way that eliminates my own biases, so that UBC is better informed of what organizations to work with and what topics to target.
Level 3 - Acceptance Criteria (The Given/When/Then format ):
1. Given that I there may be learning topics and industry experts I am not aware of, when I browse the website and make clicks and spend time on certain areas, then those decisions should be tracked and analyzed to inform what future content I may be interested in.
2. Given that I may not be familiar with all the topic options, when I access the website, then I should be presented with a random selection of videos under a variety of topics. 
3. Given UBC wants to meet as many specific producer needs as possible, when a person accesses the site, then the data informing us of their location will be saved in an anonymous format and will be linked to their interest data
